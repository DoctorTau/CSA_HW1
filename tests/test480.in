657
1430 1658 5398 -2963 2433 -7628 9207 2900 8030 -7348 1759 -2596 8363 7707 8812 -7565 -4446 -8782 4267 2877 -9087 9299 -1024 -7826 3756 -3338 -5438 1440 1794 2465 -2841 -8025 -3341 3847 -9958 -2346 -2752 -3126 1118 -7954 9177 1991 1362 -1837 -4122 -2781 -2698 -3945 7412 -5609 4318 1989 -5973 -2862 4348 -2079 -1448 6004 -6089 -9950 -2652 3272 6243 8864 267 -6151 -8546 1132 4673 5093 2043 192 -3901 8590 -7789 -1744 1885 7465 2129 -4451 -5122 -1577 -150 1174 -4374 2080 7671 6646 -7057 1212 3549 7535 1613 1923 -6972 7836 -2287 3479 -5532 -7017 7420 -7052 4414 5300 2576 8001 3750 6290 6626 -6425 -6812 9357 -6237 2644 5238 213 -1900 7903 3167 -2384 -3939 -2024 9518 -4785 -2842 5745 6622 -5620 -9685 396 1388 6780 -4408 -9329 -6719 -1246 7078 3129 -3326 638 -4337 -7054 -7750 -5745 1786 -5066 1101 -6400 6961 -1602 -7647 4014 -9970 8557 -2026 292 -5288 3661 -3497 7332 -296 -4710 -387 289 -6153 193 -3085 -5303 6664 527 -8134 -7670 -9140 2872 -3002 -4526 7067 158 1537 1693 1928 9789 4411 5241 8324 4173 -9732 4486 5417 -3080 3479 -8650 359 -8792 -5560 5032 -9942 353 2859 -9806 1358 7409 3422 878 -5180 3813 6758 -5068 4869 2172 -1689 3176 -8627 -553 -7284 -9720 9627 -3381 -2199 -3192 -5112 8174 3330 8643 6124 -4693 -7733 4928 -8643 4990 664 2278 -3896 1178 -3408 2962 -3167 3508 -9117 5131 5087 -8462 -2780 -4506 -2552 1906 7186 5245 -2879 -7049 9477 7054 -3 4552 -7104 -6207 -806 9721 -8056 -7111 1140 -6349 8904 -3788 -9804 1472 918 9192 3115 1869 9687 7711 6471 -1018 -3355 7303 1372 -6398 6996 -4892 -9570 9612 4927 8219 776 5980 6121 3086 -3941 -532 -7309 -2810 6965 -7601 -7450 -9202 -8384 -6938 9185 -9816 -9703 -7435 -5438 6228 -4942 -4170 -7091 6668 -8024 3713 6557 -4071 -811 4989 -5654 5154 -7758 2635 5271 6004 2936 7607 8723 -9068 -6908 6033 4641 4546 9448 -1851 9953 9124 -6704 7591 -9779 -7977 -700 8192 -2753 -8222 -4756 -3305 -9495 -4665 8705 -8867 7422 1433 9626 7741 -8618 4247 3107 8066 2442 -5659 -9060 -2116 9227 -5139 -5494 -4094 6700 4195 -1301 9791 -3243 2417 -5525 -8005 -7278 25 2712 82 -6821 -649 -613 -8839 6774 -9703 -3314 -8163 -8049 -2073 -6471 -3652 6701 7372 7597 7901 -7643 -8799 -1811 1691 9816 5867 -1970 2035 -973 3760 4384 7231 710 -199 3542 -7971 -5699 -9012 -8207 8979 2175 8737 2978 1829 9959 -4218 3752 -3151 595 -9475 9506 -9209 2281 -7935 3323 1931 -3467 -4540 4940 6009 5188 -4538 -2188 258 3278 6046 -231 633 8504 8172 5043 9761 -1575 216 -196 2457 780 -4897 -1377 9581 -7092 -2044 -7707 -3888 8286 6012 -8369 3298 6095 365 -877 -100 5461 3948 -9560 4744 3155 -9049 -1882 6680 2843 9803 9468 -5715 8835 6262 3454 3066 5156 785 -4355 -7292 9088 -348 -9048 -3231 -6930 -4023 -4665 -5510 126 -9836 2263 -1210 -6228 -3928 6552 -8454 -8912 4592 8119 -8417 -1179 -3573 6994 -2388 607 -9373 -72 -793 -9965 -8693 2041 5521 5920 7025 -3967 8916 -4680 1578 -4356 -4564 -8920 -1569 9514 -2672 -4054 7976 -4695 -3926 -8296 -9640 3064 -5019 2464 7561 1165 5931 -372 9788 -7790 7311 -242 5268 1182 8717 9138 -4303 1456 9988 -1330 -7503 832 2600 8676 -4062 9858 1569 6685 5672 8217 -4375 -6650 -6848 477 -7116 2747 8990 5808 -6744 2927 -4509 6464 4010 1168 3644 -2746 -3239 2944 2603 7690 -9818 6770 9687 -9571 8048 -2821 -2733 5863 7961 2179 2608 4547 -5512 -7733 -2560 -2903 2569 823 -4052 -290 4857 -9587 9030 -7966 1880 2982 -2537 -8797 -9727 -2930 -7637 -1425 9090 7113 -8608 6092 -7603 8912 -6238 -9077 -6721 5453 2165 -6743 -6406 6308 -1445 9332 -2437 5855 8377 4371 7651 1335 -8113 -5328 -7200 -458 9827 -3503 -9218 -3591 -3605 1287 -5072 -5124 6093 7194 -3527 -3663 -6341 5068 -5786 8998 3312 -8109 