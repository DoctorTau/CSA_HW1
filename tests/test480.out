-7628 -7348 -2596 -4446 -8782 -1024 -7826 -3338 -5438 -9958 -2346 -2752 -3126 -7954 -4122 -2698 -2862 -1448 -9950 -2652 -8546 -1744 -5122 -150 -4374 -6972 -5532 -7052 -6812 -1900 -2384 -2024 -2842 -5620 -4408 -1246 -3326 -7054 -7750 -5066 -6400 -1602 -9970 -2026 -5288 -296 -4710 -8134 -7670 -9140 -3002 -4526 -9732 -3080 -8650 -8792 -5560 -9942 -9806 -5180 -5068 -7284 -9720 -3192 -5112 -3896 -3408 -8462 -2780 -4506 -2552 -7104 -806 -8056 -3788 -9804 -1018 -6398 -4892 -9570 -532 -2810 -7450 -9202 -8384 -6938 -9816 -5438 -4942 -4170 -8024 -5654 -7758 -9068 -6908 -6704 -700 -8222 -4756 -8618 -9060 -2116 -5494 -4094 -7278 -3314 -3652 -1970 -9012 -4218 -4540 -4538 -2188 -196 -7092 -2044 -3888 -100 -9560 -1882 -7292 -348 -9048 -6930 -5510 -9836 -1210 -6228 -3928 -8454 -8912 -2388 -72 -4680 -4356 -4564 -8920 -2672 -4054 -3926 -8296 -9640 -372 -7790 -242 -1330 -4062 -6650 -6848 -7116 -6744 -2746 -9818 -5512 -2560 -4052 -290 -7966 -2930 -8608 -6238 -6406 -5328 -7200 -458 -9218 -5072 -5124 -5786 